### DISCUSSION TITLE
p/5671164090f846758294776642a569a0/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#36] [Announcements]
# p/5671164090f846758294776642a569a0/

出行因为身体不舒服，上周日每天都出去计划完全泡汤了。这周仅在周二去了広沢池附近，把自行车停在附近的公园里之后，就去附近走了走。基本路线是 広沢池 => 後宇多天皇 蓮華峯寺陵 => 嵯峨天皇 嵯峨山上陵 => 来迎山 称念寺。由于时间关系，附近的大覚寺没能进去看看。     下面是 google 影集  広沢池 後宇多天皇 蓮華峯寺陵 嵯峨天皇 嵯峨山上陵 称念寺  生病从上周三

https://blog.rxliuli.com/p/5671164090f846758294776642a569a0/

**Comment by ASIjacket** [comment]
大医院还是可以去的，先去个大概对症的诊所，简单说下情况，直接要求写介绍信，就可以。毕竟诊所服务性质更强一些。去过几家走的时候医生都要给我鞠躬（虽然很尴尬。

至于语言，我的日语可能比博主还差，我把所有相关的内容，pad上gpt翻译成日英，直接给给他看，基本没啥问题，大医院的医生还是很耐心的。

不造为什么博主选择京都开局，有机会也希望可以讲讲。~~我的认知里高田马场新手村的比较常见。~~

**Reply by rxliuli** [reply]
嗯，之前因为一些推友对日本医院的刻板印象比较糟糕，向老师咨询后加强了这种印象，也许下次再有什么不舒服可以去试试。另外吾辈不知道可以直接去诊所开介绍信而不需要在诊所诊断，这是新的。

京都很有名，又不是大城市，在国内的北京广州待久了有点想换换心情，在 前往日本 那期博客有说过。

**Reply by ASIjacket** [reply]
完全不需要诊断倒也不是，医生写介绍信需要一个"理由"来给接锅的医生看。

~~"你看，这病人这毛病，得你来治，お疲れ"~~

锅经甩出去了，接锅的医生最多牢骚几句，程序正义就可以了。这也是我理解上的日本人思维。

比如我"有幸"去过的很多次，诊所医生最多会做一个很简单的检查，自费部分几百○的样子。有些诊所还会把介绍信挂在首页。

京都路过多次，希望明年可以有机会实地体验下。
